python/moltres_xs.py

```diff
@@ -300,10 +299,10 @@ def generate_openmc_tallies_xml(
                 name=str(id) + "_beta",
             )
             domain_dict[id]["chi"] = mgxs.Chi(
-                domain=domain, groups=groups, name=str(id) + "_chi"
+                domain=domain, energy_groups=groups, name=str(id) + "_chi"
```

Comment:
I think because we're switching the kwarg it also means we aren't supporting older openmc versions.